### PR TITLE
Add cop to check for focused specs

### PR DESCRIPTION
**Pull request by renanborgescampos**
Detect offenses whenever focus: true arg is found

### CODE REVIEW

**Comment by danielnc** [comment]
:+1:

**Comment by andyw8** [comment]
@renanborgescampos did you mean to delete this? Seems like it would be useful.

File: lib/rubocop/cop/rspec/focus.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      #
+      #   # good
+      #   describe MyClass do
+      #   end
+      class Focus < Cop
+
+        MESSAGE = "Focused spec found.".freeze
+
+        FOCUSABLE_BLOCKS = [
+          :describe, :context, :it, :feature, :scenario
```
Comment (andyw8) [inline lib/rubocop/cop/rspec/focus.rb]: What about `specify`? Also, the pending variants such as `xit`, `xdescribe` etc.

**Comment by renanborgescampos** [comment]
@andyw8 Soon I'll open another pull request with this cop, and some code improvements.

**Pull request closed without merging**